client/a8c-for-agencies/components/page-section/backgrounds/index.tsx

```diff
@@ -2,6 +2,7 @@ import Background1Image from './background-1.svg';
 import Background2Image from './background-2.svg';
 import Background3Image from './background-3.svg';
 import Background4Image from './background-4.svg';
+import Background8Image from './background-8.svg';
```

Comment:
Sections 5, 6, and 7 are reserved for new background images according to their order.